lib/core-common/src/utils/__tests__/to-importFn.test.ts

```diff
@@ -160,6 +160,24 @@ const testCases: [string, string[], string[]][] = [
       '/Users/user/code/src/components/Icon.stories.js',
     ],
   ],
+  [
+    './Introduction.stories.tsx',
+    ['/Users/user/code/path/Introduction.stories.tsx'],
+    [
+      '/Users/user/code/Introduction.stories.tsx',
+      '/Users/user/code/src/Introduction.stories.tsx',
+      '/Users/user/code/src/Introduction.tsx',
+    ],
+  ],
+  [
+    'Introduction.stories.tsx',
+    ['/Users/user/code/path/Introduction.stories.tsx'],
+    [
+      '/Users/user/code/Introduction.stories.tsx',
+      '/Users/user/code/src/Introduction.stories.tsx',
+      '/Users/user/code/src/Introduction.tsx',
+    ],
+  ],
```

Comment:
I guess these tests are less helpful because the working dir == the config dir here. Should we change that? It's sort of unrealistic